### ISSUE TITLE
Could you share your config file for triplet loss training?

### ISSUE THREAD

**Issue by skyleft** [lyonguyen8697/tensorflow-keras-triplet-loss#1]
Hi, Could you share your config file for triplet loss training?
also the data format of annotations_file?

**Comment by neyugncol** [comment]
This is the format of the config file:
```
{
  "batch_size": 64,
  "steps_per_epoch": 750,
  "num_epochs": 20,
  "example_per_category": 5,
  "triplet_loss_margin": 1.0,
  "lr": 1e-5,
  "backbone": "resnet50",
  "backbone_weights": "imagenet",
  "feature_layers": [],
  "pooling": "max",
  "l2_normalize": true,
  "features_joint_op": "concat",
  "fc_layer": false,
  "embedding_size": 128,
  "distance_metric": "cosine",
  "image_size": 224,
  "augment_images": true,
  "image_dir": "",
  "annotations_file": "",
  "checkpoint_dir": "",

  "weight_file": "",

  "experiment_name": "",
  "comet_api_key": "",

  "verbose_training": true,
  "checkpoint_monitor": "val_accuracy",
  "checkpoint_mode": "max",
  "checkpoint_save_best_only": true,
  "checkpoint_save_weights_only": true,
  "checkpoint_verbose": true,
  "tensorboard_write_graph": true
}
```
annotations file:
```
{
  "dataset": "",
  "annotations": [
    {
      "id": int,
      "category_id": int,
      "image_file": ""
    }
  ],
  "categories": [
    {
      "id": int,
      "split": "train|val|test"
    }
  ]
}
```
